zhmccli/_cmd_console.py

```diff
@@ -159,6 +159,24 @@ def list_api_features(cmd_ctx, **options):
               default=True,
               help="Boolean indicating to accept the previous bundle level "
               "before installing the new level. Default: true")
+@click.option('--ftp-host', type=str, required=False,
+              help="The hostname for the FTP server from which the firmware "
+              "will be retrieved. Default: Firmware will be retrieved from the "
+              "IBM support site.")
+@click.option('--ftp-protocol', type=click.Choice(["ftp", "ftps", "sftp"]),
+              required=False, default="sftp",
+              help="The protocol to connect to the FTP server, if the firmware "
+              "is retrieved from an FTP server. Default: sftp.")
+@click.option('--ftp-user', type=str, required=False,
+              help="The username for the FTP server login, if the firmware "
+              "is retrieved from an FTP server.")
+@click.option('--ftp-password', type=str, required=False,
```

Comment:
just wondering: I assume that the zhmcclient also supports reading the password from STDIN, to avoid passing it directly?

Reply:
Not right now. That is a very good point. I'll add that.

Reply:
Added by specifying  `-` as the option value.